lib/slack.js

```diff
@@ -159,13 +159,23 @@ module.exports = class Slack {
     }
 
     const source = user.slack;
-    if (pullRequests.length === 0) {
+    if (pullRequests.toReview.length === 0 && pullRequests.toComplete.length === 0) {
       this.send(source, ':tada: You have no pull requests to review! :beers:');
       return;
     }
 
-    const text = 'Here are your pull requests to review:\n';
-    const attachments = pullRequests.map(Slack.createIssueAttachment);
-    this.send(source, { text, attachments });
+    if (pullRequests.toReview.length > 0) {
+      this.send(source, {
+        text: 'Here are your pull requests to review:\n',
```

Comment:
Streamline:
 - `Here are pull requests you need to *review*:\n`
 - `Here are pull requests you need to *merge or close*:\n` (or *handle*, *finish*, *finalize*)